### PR TITLE
Update Frame-VAD doc and fix onnx export

### PR DESCRIPTION
**Pull request by stevehuang52**
# What does this PR do ?

Update some VAD doc and fix onnx exportable.

Had to overwrite the `forward_for_export` function because the MLP class in `collections.common` doesn't have neural types and the param list of MLP doesn't have `encoder_output` which is used in ASR models

**Collection**: [ASR]

### CODE REVIEW

File: examples/asr/conf/vad/frame_vad_infer_postprocess.yaml [hunk 1]
```diff
@@ -1,6 +1,7 @@
 name: &name "vad_inference_postprocessing"
 
 dataset: null # Path of json file of evaluation data. Audio files should have unique names
+output: null  # Path to output directory where results will be stored
```
Comment (fayejf) [inline examples/asr/conf/vad/frame_vad_infer_postprocess.yaml:4]: how about making the name clearer?  such as output_folder

**Review by fayejf — APPROVED** [review]
minor comments

File: examples/asr/speech_classification/frame_vad_infer.py [hunk 2]
```diff
@@ -21,7 +21,8 @@
 ## Usage:
 python frame_vad_infer.py \
     --config-path="../conf/vad" --config-name="frame_vad_infer_postprocess" \
-    dataset=<Path of manifest file containing evaluation data. Audio files should have unique names>
+    dataset=<Path of manifest file containing evaluation data. Audio files should have unique names> \
+    output=<Path of output directory>
```
[the remark this answers is not in the document]
Comment (stevehuang52) [reply examples/asr/speech_classification/frame_vad_infer.py]: updated

**Review by fayejf — APPROVED** [review]
LGTM thanks

**Pull request merged**